docs/integration_resources/software/nuke.md

```diff
@@ -6,17 +6,84 @@ Shift provides a `nshift` utility script to properly bind the Shift UI to Nuke.
 
 ```python
 import sys
-sys.path.append("<path_to_your_shift_folder>")
-sys.path.append("<path_to_your_shift_folder>/thirdparty/python/Lib/site-packages")
+sys.path.append("<path_to_the_shift_installation_folder>")
+sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
 
 from shift.tools import nshift
 nshift.show()
 ```
 
+## Shift installation in Nuke
+
+To install Shift and open the Shift UI with a menu entry within Nuke, it is required to set up Shift in the user init and menu Python files from Nuke. Shift can be added to the user init and menu files from Nuke's preference folder (*<home directory>/.nuke*) or to a custom init and menu files inside a custom folder structure. If the directory where these files are stored is a path read by Nuke, they will be automatically executed on startup.
+
+
+### Init
+
+In the init.py file is required to configure the Shift basics to work in Nuke batch and interface modes. For that purpose is only required to add the Shift installation path to the environment variables. 
+
+```python
+import sys
+sys.path.append("<path_to_the_shift_installation_folder>")
+sys.path.append("<path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages")
+```

```

Comment:
Update to use code block formatting 
https://github.com/Inbibo/shift_docs/issues/60#issuecomment-2326086193